src/Modules/General/SearchComponent.js

```diff
@@ -6,9 +6,9 @@ import PlacesAutocomplete, {
 } from "react-places-autocomplete";
 
 const SearchComponent = forwardRef(({ register, errors, onSubmitHandler, searchData}, ref) => {
-  const default_zipcode = (searchData && searchData.zip_code)? searchData.zip_code : "";
+  const default_zipcode = (searchData && searchData.length > 0)? searchData : "";
```

Comment:
@seanlawrenz updated from  searchData.zip_code to just searchData
now searchData itself has only zip, but before zip attribute is inside searchData (like searchData.zip_code)